app/controllers/notes_controller.rb

```diff
@@ -178,6 +178,7 @@ def update
       @revision = @node.latest
       @revision.title = params[:title]
       @revision.body = params[:body]
+      @revision.timestamp = Time.now.to_i
```

Comment:
Revision was not updated after the edit so I had to use node.changed but that was showing edits on tag addition too so I have made a small change in revision only.Now, tag additions are not shown and edits are shown properly in the digest.